zetov/zano
# Develop

* test details & close btn

* compiled

* confirm modal

* delete "cancel" btn from aliases

* fix copy btn

* text alias fix

* confirm pop up

* lock & unlock transaction

* confirm pop up comment fix

* compiled

* rebuild html

* contact service

* rebuild html

* fix add contact + rebuild html

* allow symbols at name field diffrent from latin

* fix export import

* alias fix

* fix master pass (immigration 41 > 43)

* rebuild html

* commit

* fix

* rebuild

* fix modal width

* add type '.csv',validation empty contact list/file

* merge

* fix data.foreach & rebuild html

* fix countdown + counter of blocks

* fix logout without master pass

* fix sync block view
